src/status_im/transport/handlers.cljs

```diff
@@ -18,25 +18,36 @@
             [status-im.transport.message.v1.group-chat :as v1.group-chat]
             [status-im.data-store.transport :as transport-store]))
 
-(defn receive-message [cofx chat-id js-message]
-  (let [{:keys [payload sig]} (js->clj js-message :keywordize-keys true)
+(defn update-last-received-from-inbox
```

Comment:
I haven't quite figured how it works all together, just a rough idea, but just by looking at it:

seems like we should handle out-of-order messages? (`:inbox/last-received` > `timestamp `)

Reply:
Ah no, sorry my bad, you set it to `now`, I thought you were setting it to `timestamp`